### PR TITLE
Redesign of the starred pages

### PR DESCRIPTION
**Pull request by e34lin**
Not completely done the redesign but would appreciate any feedback or guidance.

Notes:
- Need help with the colours of each tab
- Not able to get the verified tag
- Collection length for organizations returns 0 
- Services currently show up under the workflows tab --> created a ticket for that already

### CODE REVIEW

**Comment by garyluu** [comment]
> * Need help with the colours of each tab

See https://dev.dockstore.net/search?entryType=workflows&searchMode=files for example on how to apply it. might need to refactor a bit and make a global style for the workflow, tool, and service color as I don't think they're part of the original palette.

> * Not able to get the verified tag

I think this is worth creating a separate issue for. seems /api/users/starredWorkflows and such doesn't return versions which is used to calculated whether the entry is verified 

> * Collection length for organizations returns 0

CollectionLength is transient, going to need to modify the endpoint and set it manually. Search "setCollectionsLength" in dockstore/dockstore to see how it's set. Also probably worth creating another issue for.

> * Services currently show up under the workflows tab --> created a ticket for that already

👍

**Comment by denis-yuen** [comment]
Probably worth posting a few screenshots of how this ends up looking.

**Comment by e34lin** [comment]
How it currently looks like: 
Screen Shot 2021-07-06 at 3 40 28 PM
Screen Shot 2021-07-06 at 3 41 00 PM
Screen Shot 2021-07-06 at 3 41 32 PM

File: src/app/starredentries/starredentries.component.html [hunk 1]
```diff
@@ -95,31 +227,69 @@ <h3 class="starred-header">Starred Organizations</h3>
...
+                          {{ org.collectionsLength }}
+                          <span *ngIf="org.collectionsLength != 1; else oneCollection"> Collections</span>
+                          <ng-template #oneCollection> Collection</ng-template>
+                        </span>
+                        <button mat-raised-button color="accent" class="view-button">View</button>
+                      </div>
+                    </a>
+                  </div>
+                </div>
+              </mat-card>
```
Comment (garyluu) [inline src/app/starredentries/starredentries.component.html:289]: this looks copy pasted.  "Limited duplicate code" of https://github.com/dockstore/dockstore/wiki/Dockstore-Frontend-Opinionated-Style-Guide#pr-checklist

Create a new tiny component if necessary

**Review by garyluu — CHANGES_REQUESTED** [review]
Getting horizontal (and some vertical scroll bars) when I test this out locally.

Minimize commented out code (especially since there's no explanation)

**Review by coverbeck — COMMENTED** [review]
I'm going to wait until Gary's comments are addressed before doing a more in-depth review.

Also, it would be useful, to me at least, to know what you're changing, feature-wise via the PR description, and/or a link to the ticket. Your screenshots are great, but it's hard for me to know what in the UI you are changing compared to the way it is without actually trying it out.

**Comment by e34lin** [comment]
> I'm going to wait until Gary's comments are addressed before doing a more in-depth review.
> 
> Also, it would be useful, to me at least, to know what you're changing, feature-wise via the PR description, and/or a link to the ticket. Your screenshots are great, but it's hard for me to know what in the UI you are changing compared to the way it is without actually trying it out.

Yup, that makes sense. I'll make sure to include that next time. But here is the link to the ticket: https://ucsc-cgl.atlassian.net/browse/SEAB-2905.

**Review by denis-yuen — COMMENTED** [review]
re-request review when appropriate

**Comment by e34lin** [comment]
I created tickets for both the verified tag, and collection length so they still don't work in my current PR. I was able to get the colours for each tab to work and there shouldn't be any horizontal or vertical scroll bars anymore.  

This is how it currently looks like:
Screen Shot 2021-07-29 at 1 52 50 PM
Screen Shot 2021-07-29 at 1 52 56 PM
Screen Shot 2021-07-29 at 1 53 09 PM

**Review by garyluu — COMMENTED** [review]
Lots of unrelated changes in the PR. Not sure what's up with GitHub. I forked your branch and rebased with develop. Check if https://github.com/dockstore/dockstore-ui2/pull/1320 has the changes you want. Probably work off of that.

**Comment by e34lin** [comment]
> Lots of unrelated changes in the PR. Not sure what's up with GitHub. I forked your branch and rebased with develop. Check if #1320 has the changes you want. Probably work off of that.

Yup thank you! Will request review off of that branch.

**Review by coverbeck — COMMENTED** [review]
Seems like this should be closed in favor of #1320, right? Just making sure -- not going to review unless I get confirmation.

**Review by denis-yuen — COMMENTED** [review]
ditto

**Comment by e34lin** [comment]
> Seems like this should be closed in favor of #1320, right? Just making sure -- not going to review unless I get confirmation.

yup, will be closing this now.

**Pull request closed without merging**